### DISCUSSION TITLE
性能测试对比

### DISCUSSION THREAD

**Discussion by shanshitao** [taosdata/TDengine#4638] [General]
通过代码生成数据：java com/taosdata/generator/DataGenerator -dataDir ~/testdata -numOfDevices 10000 -numOfFiles 100 -rowsPerDevice 10000
tdengineTest 使用方法： tdengineTest -dataDir ./data -numOfFiles 10 -writeClients 2 -rowsPerRequest 100 

报错：
[root@tong55 tdengine]# ./tdengineTest -dataDir ~/testdata -numOfFiles 100 -writeClients 5 -rowsPerRequest 1000
read data
---- sql: ./sqlCmd.txt
---- clients: 1
Thread 1
failed to open file ./sqlCmd.txt

源码以及安装包中并没有这个sqlCmd.txt文件，这个文件该怎么编写和使用？

**Comment by sangshuduo** [comment]
这里的 sqlCmd.txt 是指 tests/comparisonTest/tdengine 目录下的 q1.txt q2.txt q3.txt q4.txt

**Reply by dddsssggg** [reply]
> > 这里的 sqlCmd.txt 是指 tests/comparisonTest/tdengine 目录下的 q1.txt q2.txt q3.txt q4.txt
> 
> 那请问这个错误具体怎么解决呢，想要测试tdengine写入性能的话

**Reply by liuzheng3087** [reply]
写入的话加一个参数  -w 1

./tdengineTest -dataDir ~/testdata -numOfFiles 100 -writeClients 5 -rowsPerRequest 1000 -w 1
